Sources/ArcGISToolkit/Resources/en.lproj/Localizable.strings

```diff
@@ -68,205 +62,221 @@ second being a corresponding cardinal direction (north, northeast,
 
-/* No comment provided by engineer. */
+/* A message indicating that the chosen utility network element wasn't able
+ to be used as a trace starting point. */
 "Failed to set starting point" = "Failed to set starting point";
 
-/* No comment provided by engineer. */
+/* A label in reference to utility elements returned as results of a utility network
+ trace operation. */
 "Feature Results" = "Feature Results";
 
-/* No comment provided by engineer. */
+/* A field in a chart of data contained in a popup. */
 "Field" = "Field";
 
-/* No comment provided by engineer. */
+/* A label in reference to fields contained in a chart of data. */
 "Fields" = "Fields";
 
-/* A search field allowing user to filter a list of facilities by name. */
+/* A field allowing the user to filter a list of facilities by name. A
+ facility contains one or more levels in a floor-aware map or scene. */
 "Filter facilities" = "Filter facilities";
 
-/* A search field allowing user to filter a list of sites by name. */
+/* A field allowing the user to filter a list of sites by name. A site
+ contains one or more facilities in a floor-aware map or scene. */
 "Filter sites" = "Filter sites";
 
-/* No comment provided by engineer. */
+/* A hint as to what the user can search for within a search bar. */
 "Find a place or address" = "Find a place or address";
 
-/* No comment provided by engineer. */
+/* A label in reference to a fractional distance along an edge style utility network element. */
 "Fraction Along Edge" = "Fraction Along Edge";
 
-/* No comment provided by engineer. */
+/* A label in reference to function outputs returned as results of a utility network
+ trace operation. */
 "Function Results" = "Function Results";
 
-/* No comment provided by engineer. */
+/* A label in reference to media elements contained within a popup. */
 "Media" = "Media";
 
-/* No comment provided by engineer. */
+/* The mode in which the utility network trace tool is being used (either creating traces or viewing traces). */
 "Mode" = "Mode";
 
-/* No comment provided by engineer. */
+/* A label in reference to the user defined name for an individual utility network trace. */
 "Name" = "Name";
 
-/* No comment provided by engineer. */
+/* A label in reference to a specific utility network. */
 "Network" = "Network";
 
-/* No comment provided by engineer. */
+/* A label for a button to show new utility network trace configuration options. */
 "New trace" = "New trace";
 
-/* No comment provided by engineer. */
+/* A label indicating that no bookmarks are available for selection. */
 "No bookmarks" = "No bookmarks";
 
-/* No comment provided by engineer. */
+/* A statement that no utility trace configurations are available. */
 "No configurations available" = "No configurations available";
 
-/* No comment provided by engineer. */
+/* A statement that no sites or facilities with names matching a filter phrase were found. */
 "No matches found" = "No matches found";
```

Comment:
The translators enquired if this string should have a period. I believe it is okay without one but I'm open to other interpretations.

Reply:
No strong opinion here but ok to add a period as it is used in a `Text` as a message and not a title of the button etc.